lists/finding_list_0x6d69636b_machine.csv

```diff
@@ -292,11 +292,14 @@ ID,Category,Name,Method,MethodArgument,RegistryPath,RegistryItem,ClassName,Names
 1760,"Administrative Templates: Windows Components","Windows Remote Shell: Allow Remote Shell Access",Registry,,HKLM:\Software\Policies\Microsoft\Windows\WinRM\Service\WinRS,AllowRemoteShellAccess,,,,1,0,=,Medium
 2000,"Administrative Templates: Windows Components","File Explorer: Configure Windows Defender SmartScreen",Registry,,HKLM:\SOFTWARE\Policies\Microsoft\Windows\System,EnableSmartScreen,,,,1,1,=,Medium
 2001,"Administrative Templates: Windows Components","File Explorer: Configure Windows Defender SmartScreen to warn and prevent bypass",Registry,,HKLM:\SOFTWARE\Policies\Microsoft\Windows\System,ShellSmartScreenLevel,,,,Warn,Block,=,Medium
-2100,PowerShell,"Turn on PowerShell Script Block Logging",Registry,,HKLM:\Software\Policies\Microsoft\Windows\PowerShell\ScriptBlockLogging,EnableScriptBlockLogging,,,,0,1,=,Medium
-2101,PowerShell,"Turn on PowerShell Script Block Logging (Invocation)",Registry,,HKLM:\Software\Policies\Microsoft\Windows\PowerShell\ScriptBlockLogging,EnableScriptBlockInvocationLogging,,,,0,1,=,Low
-2102,PowerShell,"Turn on PowerShell Transcription",Registry,,HKLM:\Software\Policies\Microsoft\Windows\PowerShell\Transcription,EnableTranscripting,,,,0,1,=,Low
-2103,PowerShell,"Disable PowerShell version 2",WindowsOptionalFeature,MicrosoftWindowsPowerShellV2,,,,,,Enabled,Disabled,=,Medium
-2104,PowerShell,"Disable PowerShell version 2 (root)",WindowsOptionalFeature,MicrosoftWindowsPowerShellV2Root,,,,,,Enabled,Disabled,=,Medium
+2100,PowerShell,"Turn on PowerShell Module Logging",Registry,,HKLM:\Software\Policies\Microsoft\Windows\PowerShell\ModuleLogging,EnableModuleLogging,,,,0,1,=,Low
```

Comment:
I read https://medium.com/@ammadb/powershell-logging-module-logging-vs-script-block-logging-7aa74bf66261 when I was trying to decide whether it would be helpful to add PowerShell Module Logging since you already check for Script Block Logging. While I was researching the differences between Module Logging and Script Block Logging, I found https://www.mandiant.com/resources/blog/greater-visibilityt. In that article, it elaborates, "Where possible, Mandiant recommends enabling all three log sources: module logging, script block logging and transcription. Each of these sources records unique data valuable to analyzing PowerShell activity." When I ran tests with Script Block Logging and Module Logging enabled, the Event ID 4103 (Windows PowerShell Module Logging) events appeared to provide a small amount of additional information (i.e. the name of the PowerShell module) which was executed. In situations where forensics are needed to understand malware that uses PowerShell, I can understand the desire to add Module Logging. While less important if Script Block Logging is enabled, I felt also Turning PowerShell Module Logging on would provide the best visibility on PowerShell commands. If there was a severity lower than "Low", I would have used that.

Reply:
Can you keep the ID's of the previous PowerShell checks and increase the ID's for module logging?